pkgs/desktops/gnome/extensions/pano/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, mkYarnModules
+, fetchFromGitHub
+, atk
+, cogl
+, glib
+, gnome
+, gtk3
+, libgda
+, pango
+, nodePackages
+}:
+
+let
+  version = "v3";
+  uuid = "pano@elhan.io";
+in
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-pano";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "oae";
+    repo = "gnome-shell-pano";
+    rev = version;
+    hash = "sha256-LVEtxh/+Zjx0TXcP2+7SeW7jKDw9baoea35MQpNfnmQ=";
+  };
+
+  nativeBuildInputs = [
+    nodePackages.rollup
+  ];
+
+  buildInputs = [
+    atk
+    cogl
+    glib
+    gnome.gnome-shell
+    gnome.mutter
+    gtk3
+    libgda
+    pango
+  ];
+
+  nodeModules = mkYarnModules {
+    inherit pname version;
+    name = "${pname}-modules-${version}";
+    packageJSON = ./package.json;
+    yarnLock = ./yarn.lock;
+    yarnNix = ./yarn.nix;
+  };
+
+  buildPhase =
+    let
+      dataDirPaths = lib.concatStringsSep ":" [
+        "${atk.dev}/share/gir-1.0"
+        "${gnome.gnome-shell}/share/gnome-shell"
+        "${gnome.mutter}/lib/mutter-10"
+        "${gtk3.dev}/share/gir-1.0"
+        "${libgda}/share/gir-1.0"
+        "${pango.dev}/share/gir-1.0"
+      ];
+    in
+    ''
+      runHook preBuild
+
+      ln -sv "${nodeModules}/node_modules" node_modules
+      XDG_DATA_DIRS="$XDG_DATA_DIRS:${dataDirPaths}" \
+          node_modules/@gi.ts/cli/bin/run config --lock
+      node_modules/@gi.ts/cli/bin/run generate
+      rollup -c
+      glib-compile-schemas ./resources/schemas --targetdir=./dist/schemas/
+
+      runHook postBuild
+    '';
+
+  passthru = {
+    extensionUuid = uuid;
+    extensionPortalSlug = "pano";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p "$out/share/gnome-shell/extensions/${uuid}"
+    cp -r dist/* "$out/share/gnome-shell/extensions/${uuid}/"
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Next-gen Clipboard Manager for Gnome Shell";
+    license = licenses.gpl2;
```

Comment:
`gpl2` is unclear, use `gpl2Plus` or `gpl2Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
fixed